mewarpx/mewarpx/utils_store/init_restart_util.py

```diff
@@ -89,6 +90,98 @@ def clean_old_checkpoints(checkpoint_directory="diags",
         shutil.rmtree(dirpath)
 
 
+def _eval_corrupt_checkpoint(checkpoint_dir, checkpoint):
+    """Determine if a checkpoint appears to be corrupt by checking if
+    fluxdata.ckpt is present.
```

Comment:
```suggestion
def _eval_checkpoint_validity(checkpoint_dir, checkpoint):
    """Determine if a checkpoint appears to be valid by checking if
    fluxdata.ckpt is present.
```

Almost always better to phrase names in the affirmative, especially since the function is actually returning the affirmative.